spa/src/components/Error/KnownErrors/index.tsx

```diff
@@ -39,14 +59,103 @@ export const ErrorForSSO = ({ orgName, accessUrl, resetCallback, onPopupBlocked
 	</div>
 </>;
 
-export const ErrorForNonAdmins = ({ orgName, adminOrgsUrl }: { orgName?: string; adminOrgsUrl: string; }) => <div css={paragraphStyle}>
-	Can't connect, you're not the organization owner{orgName && <span> of <b>{orgName}</b></span>}.<br />
-	Ask an <a css={linkStyle} onClick={() => {
-	// TODO: Need to get this URL for Enterprise users too, this is only for Cloud users
-		popup(adminOrgsUrl);
+export const ErrorForNonAdmins = ({ orgName, adminOrgsUrl, onPopupBlocked, deferredInstallationOrgDetails , hostUrl}: {
+	orgName?: string;
+	adminOrgsUrl: string;
+	onPopupBlocked: () => void;
+	deferredInstallationOrgDetails: DeferredInstallationUrlParams;
+	hostUrl?: HostUrlType;
+}) => {
+	const [isOpen, setIsOpen] = useState<boolean>(false);
+	const [isLoading, setIsLoading] = useState<boolean>(false);
+	const [deferredInstallationUrl, setDeferredInstallationUrl] = useState<string | null>(null);
+
+	const getOrgOwnerUrl = async () => {
+		// TODO: Need to get this URL for Enterprise users too, this is only for Cloud users
+		const win = popup(adminOrgsUrl);
+		if (win === null) onPopupBlocked();
 		analyticsClient.sendUIEvent({ actionSubject: "checkOrgAdmin", action: "clicked"}, { type: "cloud" });
-	}}>organization owner</a> to complete this step.
-</div>;
+	};
+
+	const getDeferredInstallationUrl = async () => {
+		if (!isOpen) {
+			try {
+				setIsOpen(true);
+				setIsLoading(true);
+				const response = await Api.deferral.getDeferredInstallationUrl({
+					gitHubInstallationId: deferredInstallationOrgDetails?.gitHubInstallationId ,
+					gitHubOrgName: deferredInstallationOrgDetails?.gitHubOrgName
+				});
+				setDeferredInstallationUrl(response.data.deferredInstallUrl);
+				// TODO: Create events in amplitude
+			} catch(e) {
+				// TODO: handle this error in UI/Modal ?
+				console.error("Could not fetch the deferred installation url: ", e);
+			} finally {
+				setIsLoading(false);
+			}
+		}
+	};
+
+	const closeModal = () => {
+		setIsOpen(false);
+		setDeferredInstallationUrl(null);
+	};
+	return (
+		<div css={paragraphStyle}>
+			You’re not an owner for this organization. To connect:
+			<ol css={olStyle}>
+				<li>
+					<a css={linkStyle} onClick={getOrgOwnerUrl}>
+						Find an organization owner.
+					</a>
+				</li>
+				<li>
+					<a css={linkStyle} onClick={getDeferredInstallationUrl}>
+						Send them a link and ask them to connect.
```

Comment:
Can you double check with design? I feel the word "Send" is a bit too strong, user might hestigate to click it.
A "lighter" word will be like "Generate" or "Prepare"